src/encoded/search_views.py

```diff
@@ -711,3 +714,94 @@ def top_hits(context, request):
         ]
     )
     return fr.render()
+
+
+@view_config(route_name='rnaget', request_method='GET', permission='search')
+def rnaget(context, request):
+    genes = request.params.get('genes')
+    units = request.params.get('units', 'tpm')
+    sort  = request.params.get('sort')
+    page  = request.params.get('page')
+
+    filters = {
+        'assayType': request.params.get('assayType'),
+        'annotation': request.params.get('annotation')
+    }
+
+    filters_data = [{'field': 'type', 'term': 'Rna Get', 'remove': '/rnaget'}]
+
+    for f in filters:
+        if filters.get(f):
+            query_string = []
+
+            for param in request.params:
```

Comment:
we are looking to remove the query string and its value from the endpoint, it can be placed in any position and also multiple times. This reproduces the snovault behavior.